dll/win32/shell32/brfolder.cpp

```diff
@@ -976,30 +1096,16 @@ BrFolder_OnChangeEx(
         case SHCNE_DRIVEADD:
         case SHCNE_MKDIR:
         case SHCNE_CREATE:
-        {
-            // FIXME
-            break;
-        }
         case SHCNE_DRIVEREMOVED:
         case SHCNE_RMDIR:
         case SHCNE_DELETE:
-        {
-            HTREEITEM hRoot = TreeView_GetRoot(info->hwndTreeView);
-            for (;;)
-            {
-                HTREEITEM hItem = BrFolder_FindItemByPidl(info, pidl0, hRoot);
-                if (!hItem)
-                    break;
-                TreeView_DeleteItem(info->hwndTreeView, hItem);
-            }
-            break;
-        }
         case SHCNE_RENAMEFOLDER:
         case SHCNE_RENAMEITEM:
         case SHCNE_UPDATEDIR:
         case SHCNE_UPDATEITEM:
         {
-            // FIXME
+            // FIXME: Optimize for speed
```

Comment:
Besides the FIXME comment suggested by @whindsaks you may add a bit more explanation about what aspects can be optimized and some ideas about how.

Reply:
Improved that comment.